admin/database/postgres/migrations/0012.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE user_auth_tokens ADD COLUMN used_on TIMESTAMPTZ DEFAULT now() NOT NULL;
+ALTER TABLE service_auth_tokens ADD COLUMN used_on TIMESTAMPTZ DEFAULT now() NOT NULL;
+ALTER TABLE users ADD COLUMN active_on TIMESTAMPTZ DEFAULT now() NOT NULL;
```

Comment:
Can we also add `UPDATE` statements setting these fields to the `updated_on` value for existing rows? Just to make existing rows have more conservative values